Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -185,6 +185,36 @@ extension BrowserViewController: WKNavigationDelegate {
     let isPrivateBrowsing = PrivateBrowsingManager.shared.isPrivateBrowsing
     let tab = tabManager[webView]
 
+    let domainForRequestURL = Domain.getOrCreate(
+      forUrl: url,
+      persistent: !isPrivateBrowsing
+    )
+
+    // Debouncing logic
+    // Handle debouncing for main frame only and only if the site (etld+1) changes
+    // We also only handle `http` and `https` requests
+    if url.isWebPage(includeDataURIs: false),
+       let currentURL = tab?.webView?.url, currentURL.baseDomain != url.baseDomain,
```

Comment:
nit: is this a swift convention, to do variable assignment a) on the same line, and b) w/in a conditional? 

If its a convention, then understood, but to a non-swift person, this line is difficult to parse for me and seems like a possible area for confusion